src/amoc_api_scenario_handler.erl

```diff
@@ -21,18 +21,75 @@
 
 -spec trails() -> trails:trails().
 trails() ->
+    Id =
+    #{name => <<"id">>,
+      in => path,
+      description => <<"Scenario id">>,
+      type => string,
+      required => true},
+
+    RequestBody =
+    #{name => <<"request body">>,
+      in => body,
+      description => <<"request body (as json)">>,
+      required => true,
+      schema =>
+      #{type => <<"object">>,
+        required => [<<"users">>],
+        properties =>
+        #{users => #{<<"type">> => <<"integer">>,
+                           <<"description">> => <<"Number of users to start">>
+                         }
+        }
+      }
+    },
+
+    ResponseBodyGet =
+    #{<<"200">> =>
+      #{description => <<"response object">>,
+        schema =>
+        #{type => <<"object">>,
+          required => [<<"scenario_status">>],
+          properties =>
+          #{scenario_status => #{<<"type">> => <<"string">>,
+                                 <<"description">> => <<"loaded | running | finished">>
```

Comment:
Line 55 is too long:                                  <<"description">> => <<"loaded | running | finished">>.